src/views/Scenario/labels.js

```diff
@@ -78,3 +78,28 @@ export const getReportLabels = (t) => ({
     ),
   },
 });
+export const getShareScenarioDialogLabels = (t, currentScenarioName) => ({
+  button: {
+    title: t('commoncomponents.dialog.share.button.label', 'Share'),
+    tooltip: t('commoncomponents.dialog.share.button.tooltip', 'Modify access'),
+  },
+  dialog: {
+    title: t('commoncomponents.dialog.share.dialog.title', 'Share ') + currentScenarioName,
+    addPeople: t('commoncomponents.dialog.share.dialog.select.addPeople', 'Add people'),
+    cancel: t('commoncomponents.dialog.share.dialog.buttons.cancel', 'Cancel'),
+    share: t('commoncomponents.dialog.share.dialog.buttons.share', 'Share'),
+    userSelected: t('commoncomponents.dialog.share.dialog.select.userSelected', 'Selected user'),
+    workspace: t('commoncomponents.dialog.share.dialog.editor.workspace', 'Workspace'),
```

Comment:
Maybe the generic component should not mention `workspace` in the labels keys to stay generic enough. Shouldn't it be the same label as `defaultRole` ?

Reply:
But wouldn't it be too repetitive to put "General access" twice ? To avoid "Workspace" maybe we can use "Default role" or "Default access level"?